apps/frontend/src/libs/components/header/header.tsx

```diff
@@ -0,0 +1,16 @@
+import { Avatar, Logo } from "~/libs/components/components.js";
+
+import styles from "./styles.module.css";
+
+const Header: React.FC = () => {
+	return (
+		<header className={styles["header"]}>
+			<div className={styles["header-inner"]}>
+				<Logo hasLink={true} />
```

Comment:
Let's use a shorthand here
```suggestion
				<Logo hasLink />
```